docs/how_tos/i18n.rst

```diff
@@ -184,8 +184,26 @@ Create your pipeline job on Jenkins
 
      * Note that the Jenkins machine uses UTC.
 
+#. Test your change locally.
+
+   * Clone https://github.com/openedx/ecommerce-scripts.
+
+   * Create virtualenv for python.
+
+   * Run ``pip install -r transifex/requirements/microfrontend-common.txt``.
+
+   * Run the script that you try to create the jobs.
+
+     * Examples: `python transifex/pull.py https://github.com/edx/edx-platform.git edx/sustaining-team`. You might be required to setup environment variable ``GITHUB_ACCESS_TOKEN`` with your personal github account.
+
+       * For `push_translations`, you might also need transifex authentication. It is necessary to push first because you need transifex to be created before pull. If there are problems with transifex auth or priviledge, you should proceed to the next step and run push job once. 
+
+       * For `pull_translations`, it should try to create a pull request and merge it to the repo.
+
 #. Open a pull request with your change, then `open a devops ticket <https://openedx.atlassian.net/servicedesk/customer/portal/3>`_ requesting them to review your pull request and rerun the seed job. Only devops can merge pull requests in this repo.
 
+   * Alternatively, you can create the job by build https://tools-edx-jenkins.edx.org/job/Seed%20Jobs/job/Translation%20Seed%20Jobs/. (You need to connect to the edX VPN)
```

Comment:
It's good you listed the edX jenkins translation jobs. But that should just be an example for edx.org. For other openedx community members, you need to say something about creating your own translation automated jobs
This is an open source repository, which means ANYONE on the internet would know this. Frankly, you have already exposed this URL to the WWW on a PR which can be read by ANYONE. Since the URL is behind edX VPN, that should be OK. That said, please be aware you are on open source repo and if you expose secrets, that's a cybersecurity instance.

Reply:
Because similar url is being share below this line. I was just assuming it was okay. However, I will keep open community in mind in the future.